features/google/src/model/analyses/ministories/activity-by-products.js

```diff
@@ -0,0 +1,17 @@
+import { RootAnalysis } from "@polypoly-eu/poly-analysis";
+import analysisKeys from "../analysisKeys";
+
+export default class ActivityByProductAnalysis extends RootAnalysis {
+    async analyze({ dataAccount: googleAccount }) {
+        const activities = googleAccount.activities;
+        const activitiesByProducts = {};
+        activities.forEach((activity) => {
+            if (!activitiesByProducts[activity.productName])
```

Comment:
this operation is repeated in a bunch of the analyses, maybe we can make a function for it

Reply:
Saved me one comment here, buddy :+1: